lib/vertica/protocol/frontend/startup.rb

```diff
@@ -8,12 +8,22 @@ def initialize(user, database, options = nil)
         @user     = user
         @database = database
         @options  = options
+        @type     = "vertica-rb"
+        @pid      = Process.pid.to_s
+        @platform = Gem::Platform.local.to_s
+        @version  = Vertica::VERSION
+        @label    = @type+"-"+@version+"-"+SecureRandom.hex(10) 
```

Comment:
Here's the JDBC for reference:

```
$ java -cp ./vertica-jdbc-7.1.2-0.jar:./ javatest 
Connecting to database...

vsql => select client_type, client_os, client_pid, client_label, client_version from sessions;
-[ RECORD 2 ]--+-----------------------------------------------------
client_type    | JDBC Driver
client_os      | Linux 4.8.0-22-generic amd64
client_pid     | 12378
client_label   | jdbc-07.01.0200-349d3ee6-9569-4a9b-a050-a8bb0ee878dd
client_version | 07.01.0200
```

To me it looked like label = type+version+uuid, so that's what I implemented for `vertica-python` as well:

```
-[ RECORD 2 ]--+-----------------------------------------------------------
client_type    | vertica-python
client_os      | Linux-4.8.0-22-generic-x86_64-with-Ubuntu-16.10-yakkety
client_pid     | 13007
client_label   | vertica-python-0.6.11-aa60a6b0-c1a3-11e6-84aa-843a4b1dd280
client_version | 0.6.11
```

This doesn't seem to be defined or enforced in any Vertica documentation/forum so "guessing what JDBC does" was my only option.